### DISCUSSION TITLE
User should see in the PR UI that Actions do not run on a PR with conflicts

### DISCUSSION THREAD

**Discussion by aerben** [community/community#14388] [Actions] [answered]
When the Branch of a PR has conflicts with the base, GH Actions does not run workflows triggered by the PR.
This makes sense. However, the user should be informed about this in the UI where it says that the branch has conflicts that prevent merging.

**Comment by ethomson** [comment, marked as answer]
I think that this is a more general problem than just Actions - no CI provider would be able to run a build against a conflicting PR. I'll let the pull requests team know that they may have an opportunity to improve the verbiage here. Thanks for the feedback.

**Reply by aerben** [reply]
Yeah, it totally makes sense, but basically all new team members ask this same question at some point in time - why is my action not triggering? Because there is a conflict. If there was a hint in the UI they might not wonder about this.

**Reply by jjb** [reply]
> no CI provider would be able to run a build against a conflicting PR

all CI providers happily run builds against conflicting PRs because they don't check that, they just check out the branch and run it
